create_db.py

```diff
@@ -21,17 +21,23 @@
 print ("Database opened successfully!")
 cursor = connection.cursor()
 
-cursor.execute("CREATE TABLE id (id integer primary key, name text, ipaddress text, office text)")
+cursor.execute("CREATE TABLE IF NOT EXISTS id (id integer primary key, name text, ipaddress text, office text)")
```

Comment:
To create the table only if it doesn't exist, otherwise it will try to create it each time the code is executed and will generate unnecessary errors.